src/TulsiGenerator/Scripts/bazel_build.py

```diff
@@ -831,7 +830,7 @@ def ReaderThread(file_handle, out_buffer):
     process = subprocess.Popen(command,
                                stdout=subprocess.PIPE,
                                stderr=subprocess.STDOUT,
-                               bufsize=1)
```

Comment:
why did this one change? doesn't look like the behavior changed between versions?

Reply:
To workaround this warning:

```
subprocess.py:848: RuntimeWarning: line buffering (buffering=1) isn't supported in binary mode, the default buffer size will be used
```

Reply:
bufsize=1 requires universal_newlines=True. We're also working on this internally